nginx/conf/nginx.conf

```diff
@@ -99,6 +99,7 @@ http {
 	# Virtual Host Configs
 	##
 
-	include /etc/nginx/conf.d/*.conf;
+	include /etc/nginx/conf.d/http.conf;
```

Comment:
If `http.conf` is to be written by user and `default.conf` by EasyEngine, then `default.conf` should be loaded before `http.conf`